pom.xml

```diff
@@ -27,7 +27,7 @@
 
         <!-- TDS Version Props -->
         <tds-common.version>0.0.15</tds-common.version>
-        <tds-exam-client.version>0.0.12</tds-exam-client.version>
+        <tds-exam-client.version>0.0.15-SNAPSHOT</tds-exam-client.version>
```

Comment:
Snapshot version will be removed before merge - once the ExamService PR is approved/released